.circleci/config.yml

```diff
@@ -6,7 +6,6 @@ stdenv: &stdenv
     GOCACHE: &gocache /tmp/go-build
     IMAGE: &image quay.io/crio/crio-build-amd64-go1.15:master-1.5.0
     IMAGELEGACY: &imagelegacy quay.io/crio/crio-build-amd64-go1.14:master-1.5.0
-    IMAGE386: &image386 quay.io/crio/crio-build-386-go1.15:master-1.5.0
```

Comment:
I'd say that we drop the 32bit binary, because this will not work with the Rust lib. We could also build a 32bit version of the rust library, but I do not see a strong reason to stick to 32bit binaries.

Reply:
I do know of some raspbian users and the 64 bit versions are still fairly recent.... I would vote we keep it

Reply:
That's ARM only, right? Phew…we probably have to build the static library for all of those architectures then… :thinking:

Reply:
yeah I believe only ARM even thinks about 32 bit these days :)

Reply:
Okay, what do we wanna do now? Build the library here in this repository? It would cause the biggest changes to our CI but seems the only reason to not put static libraries into this repo…

Reply:
I'll add the toolchain to our CI.